src/components/QrCodeForm.vue

```diff
@@ -0,0 +1,208 @@
+
+  if(sellerPixKey && transactionValue){
+    var body_req = {
+      e2e_id: value,
+      pix_key: sellerPixKey,
+      pix_value: transactionValue,
+    };
+    var resp = await api.post("http://localhost:8000/validate_pix", body_req)
+      .catch((reason: Error) => {
+        console.log('entrou no erro', reason);
+        pixIsValid.value = 2;
+        return;
+      });
+    console.log("🚀 ~ file: QrCodeForm.vue:47 ~ validatePix ~ resp", resp);
+    console.log(resp.status);
+
+    pixIsValid.value = 1;
+  } else {
+    pixIsValid.value = 0;
+  }
+};
+</script>
+
+<template>
+  <div class="page">
+    <div class="text-container">
+      <span class="text font-extrabold text-2xl max-w-[30rem]">
+        Utilize o QR Code ou copie o código para realizar o Pix
+      </span>
+      <span class="text font-medium text-md max-w-[28rem]">
+        Após realizar o Pix no banco de sua preferência, insira o código de
+        autenticação para enviar a transação para a rede.
+      </span>
+    </div>
+    <div class="blur-container max-w-[28rem] text-black">
+      <div
+        class="flex-col items-center justify-center flex w-full bg-white p-8 rounded-lg"
+      >
+        <img :src="qrCode" class="w-48 h-48" />
+        <span class="text-center font-bold">Código pix</span>
+        <span class="text-center text-xs">
+          {{ pixTarget }}
+        </span>
+        <img
+          alt="Copy PIX code"
+          src="@/assets/copyPix.svg"
+          width="16"
+          height="16"
+          class="pt-2 mb-5 cursor-pointer"
+        />
+        <span class="text-center text-xs text-start">
+          <strong>ATENÇÃO!</strong> A transação só será processada após inserir
+          o código de autenticação. Caso contrário não conseguiremos comprovar o
+          seu depósito e não será possível transferir os tokens para sua
+          carteira. Confira aqui como encontrar o código no comprovante.
+        </span>
+      </div>
+      <div
+        class="flex-col items-center justify-center flex w-full bg-white p-5 rounded-lg px-5"
+      >
+        <input
+          type="text"
+          placeholder="Digite o código do comprovante PIX"
+          @input="debounce(handleInputEvent, 500)($event)"
+          class="text-md box-border w-full box-border p-2 h-6 mb-2"
+        />
+        <div class="flex flex-col w-full" v-if="pixIsValid == 2">
+          <div class="custom-divide"></div>
+          <div class="flex items-center h-8">
+            <img
+              alt="Invalid Icon"
+              src="@/assets/invalidIcon.svg"
+              width="14"
+              class="cursor-pointer align-middle inline-block"
+            />
+            <span class="px-1 text-red-500 font-normal text-xs"
+              >Código inválido. Por favor, confira e tente novamente.</span
+            >
+          </div>
+        </div>
+        <div class="flex flex-col w-full" v-else-if="pixIsValid == 1">
+          <div class="custom-divide"></div>
```

Comment:
Custom divide is the same in showing error or success.
I think that the v-if could be in the <div class="flex flex-col w-full" v-if="pixIsValid == 1".
That's the div that has some changes.